internal/shellgen/flake_input.go

```diff
@@ -47,28 +47,33 @@ func (f *flakeInput) PkgImportName() string {
 	return f.Name + "-pkgs"
 }
 
-func (f *flakeInput) BuildInputs() ([]string, error) {
-	var err error
-	attributePaths := lo.Map(f.Packages, func(pkg *devpkg.Package, _ int) string {
-		attributePath, attributePathErr := pkg.FullPackageAttributePath()
-		if attributePathErr != nil {
-			err = attributePathErr
-		}
-		return attributePath
-	})
-	if err != nil {
-		return nil, err
+type buildInput struct {
+	AttrPath   string
+	PatchGlibc bool
+}
+
+func (f *flakeInput) BuildInputs() ([]buildInput, error) {
```

Comment:
You should be able to revert this as well once you do the individual flake approach since `PatchGlibc` is not needed in the top level flake.